### DISCUSSION TITLE
Method invocation failed because [PSADT.UserInterface.Services.AppProcessInfo] does not contain a method named 'Refresh'.

### DISCUSSION THREAD

**Discussion by jrollie** [PSAppDeployToolkit/PSAppDeployToolkit#1167] [Q&A]
Trying to figure out why I am getting this error when attempting to run an install of an already opened application. After I hit "close & install", it throws this error. Deployment works fine if the application is not open.

**Comment by jrollie** [comment]
Just adding more context. Seems to only be happening with v4 installs with Nitro currently. I did test some other apps and they were able to close/install successfully. I have no issues with the v3 deployment for Nitro closing.

**Comment by mjr4077au** [comment]
This should be opened as a bug, not a discussion. You'll also need to provide the on-screen output as there's not much useful mentioned here.
